### DISCUSSION TITLE
Export

### DISCUSSION THREAD

**Discussion by notDavid** [pieces-app/support#187] [Q&A] [answered]
Hi there, just checking out the product, and one important question that come up: is it possible to export (all) code snippets somehow?

(talking specifically about the code snippet content / code - so not necessarily all the meta-data...)

Please advise,
thank you!

**Comment by rosie-at-pieces** [comment, marked as answer]
It's not something we have in the UI at the moment but we can do manually for you if you'd like - but definitely a good feature for us to add!

**Comment by joaoponceleao** [comment]
Lack of export is the main reason I haven't actually tried pieces yet. I am very weary of vendor lock in.
